src/pymmcore_plus/mda/handlers/_ome_zarr_writer.py

```diff
@@ -278,9 +278,7 @@ def new_array(self, key: str, dtype: np.dtype, sizes: dict[str, int]) -> zarr.Ar
         self._group.attrs["multiscales"] = scales
         ary.attrs["_ARRAY_DIMENSIONS"] = dims
         if seq := self.current_sequence:
-            ary.attrs["useq_MDASequence"] = to_builtins(
-                seq.model_dump(exclude_unset=True)
-            )
+            ary.attrs["useq_MDASequence"] = to_builtins(seq)
```

Comment:
@wl-stepp, this would have been one way to fix #375... 
the tldr; is that we already *do* have a stdlib pre-serializer called `to_builtins`, and it was already capable of handling pydantic models well.  So calling `model_dump` here (without using `mode='json'`) was one of the problems